img/b/bereal.com.svg

```diff
@@ -0,0 +1 @@
M131.188 278.454c-.896-21.723 9.796-38.169 33.505-37.239 19.802.83 27.29 15.408 28.369 33.558l-.02 5.756h-21.821c-25.084 0-23.08-.514-20.356 5.038 2.674 5.295 6.683 7.967 13.211 8.79 5.551.668 11.205-.309 17.374-3.084l2.056-.925 2.416 6.014 2.416 6.014c-5.426 4.686-13.221 6.857-21.126 6.938-19 .194-35.186-10.542-36.024-30.86zm151.173.072c-.896-21.723 9.797-38.169 33.505-37.239 19.802.83 27.291 15.408 28.37 33.558l-.02 5.757h-21.822c-25.084 0-23.08-.514-20.355 5.037 2.673 5.295 6.682 7.968 13.21 8.79 5.552.669 11.206-.308 17.374-3.084l2.056-.925 2.416 6.014 2.416 6.014c-5.426 4.686-13.221 6.857-21.126 6.938-19 .194-35.186-10.542-36.024-30.86zm45.379-10.004c0-1.542-2.108-6.271-3.65-8.122-2.57-3.136-5.14-4.214-9.972-4.163-3.136 0-4.575.257-6.168 1.079-3.598 1.902-6.425 5.86-7.197 10.075l-.359 1.902h13.673c11.668 0 13.673-.103 13.673-.771zm117.92 21.26h16.705v17.477H445.66v-17.477zm-27.501-72.992h17.991v90.983h-17.991V216.79zm-211.779-.103l17.888.258c6.828.104 14.699-.239 22.128 1.127 15.438 2.839 25.9 13.07 24.006 29.905-1.553 13.807-11.528 19.038-15.189 20.782 0 .205 4.934 8.995 11 19.532l10.949 19.225c-2.5.045-16.881.4-17.581.051-.308-.102-4.831-7.967-10.022-17.476l-9.848-17.128c-5.484.749-10.831.37-16.368.37v34.44H206.38v-91.086zm46.006 28.632c-.064-8.421-4.774-12.96-13.222-13.714-5.202-.464-10.591-.442-15.821-.33v27.94c7.186.074 17.557.763 23.687-3.102 3.181-2.006 5.307-5.308 5.356-10.794zM52.172 216.687l20.201.258c13.084.149 27.624-1.283 36.052 6.655 6.787 6.393 9.687 18.376 5.6 26.878-1.356 2.82-3.452 5.498-6.338 8.154l1.439 1.028c2.159 1.131 6.631 5.294 8.328 7.762 4.831 7.042 5.037 19.173.359 27.089-3.083 5.294-9.561 9.715-17.27 11.874-3.65.977-5.45 1.079-26.113 1.285l-22.258.154v-91.137zm42.716 75.922c6.168-2.159 8.43-5.346 8.43-11.823 0-6.117-1.953-9.252-7.197-11.668-2.416-1.131-3.341-1.182-14.752-1.336l-12.234-.206v25.804h11.771c9.253 0 12.234-.154 13.982-.771zm2.416-50.734c-.154-4.884-.823-6.374-3.958-8.482-2.519-1.696-6.014-2.21-15.473-2.21h-8.738v22.72l9.921-.206c9.079-.141 18.669.175 18.248-11.822zm268.065 66.155a21.822 21.822 0 01-8.163-3.761c-9.248-6.87-10.536-20.406-1.526-28.135 1.259-1.079 2.641-2.019 4.034-2.698 4.729-2.365 8.739-3.135 19.379-3.753 12.902-.719 11.823-.359 11.823-4.215-.052-5.551-2.262-8.79-6.939-10.074-4.987-1.389-10.435-.155-13.314 3.032-.977 1.131-1.953 2.622-2.159 3.342l-.309 1.284h-15.832l.309-2.159c.874-5.962 5.346-11.977 11.308-15.163 5.089-2.725 8.723-3.698 16.844-3.698 13.391-.087 22.176 3.694 26.797 16.137l.154 24.828.155 24.776h-16.5v-4.883c0-2.673-.206-4.884-.411-4.884-.258 0-1.183 1.028-2.056 2.211-4.524 6.373-14.959 9.817-23.594 7.813zm16.808-12.697c5.5-1.696 8.739-6.271 8.739-12.336v-3.496h-2.673c-1.491 0-5.346.257-8.481.514-8.945.72-12.44 2.776-12.851 7.556-.565 6.837 6.528 10.435 15.266 7.762zm-205.61-26.883c0-1.543-2.108-6.272-3.65-8.122-2.57-3.136-5.141-4.215-9.972-4.164-3.136 0-4.575.257-6.168 1.08-3.598 1.902-6.425 5.86-7.197 10.075l-.359 1.902h13.673c11.668 0 13.673-.103 13.673-.771z"/></svg>
```

Comment:
The logo you uploaded seems to be a fan-made version from [brandfetch](https://brandfetch.com/bere.al).

I suggest you use this one instead:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><rect width="100" height="100" fill="#121212" ry="20"/><path fill="#fff" d="M31.67 45.755c1.977 0 3.43.654 4.357 1.961.928 1.307 1.392 2.853 1.392 4.635v1.04h-8.507c.156.83.538 1.493 1.145 1.986.607.494 1.405.74 2.393.74.573 0 1.089-.073 1.548-.22.46-.147.993-.36 1.6-.637l.989 2.441c-.59.45-1.306.78-2.146.987-.842.208-1.531.312-2.069.312-2.185 0-3.906-.598-5.164-1.792-1.257-1.195-1.886-2.796-1.886-4.805 0-2.008.568-3.618 1.704-4.83 1.136-1.212 2.684-1.818 4.644-1.818zm29.449 0c1.977 0 3.43.654 4.358 1.961.927 1.307 1.391 2.853 1.391 4.635v1.04h-8.507c.157.83.538 1.493 1.145 1.986.607.494 1.405.74 2.393.74.573 0 1.088-.073 1.548-.22.46-.147.993-.36 1.6-.637l.989 2.441c-.59.45-1.306.78-2.146.987-.842.208-1.531.312-2.069.312-2.185 0-3.906-.598-5.164-1.792-1.257-1.195-1.886-2.796-1.886-4.805 0-2.008.568-3.618 1.704-4.83 1.136-1.212 2.684-1.818 4.644-1.818zm12.918.14c3.245 0 5.342 1.672 5.342 4.198v8.595h-3.421v-1.87h-.07c-.715 1.285-2.05 2.069-3.702 2.069-2.519 0-4.217-1.579-4.217-3.79v-.022c0-2.28 1.769-3.614 4.885-3.813l3.104-.187v-.771c0-1.112-.726-1.802-2.062-1.802-1.277 0-2.05.597-2.214 1.404l-.023.105H68.53l.012-.14c.187-2.316 2.178-3.977 5.494-3.977ZM90 55.224v3.463h-3.469v-3.462ZM16.608 41.029c1.717 0 3.113.377 4.188 1.13 1.075.754 1.613 2.022 1.613 3.805 0 .658-.152 1.25-.455 1.779a4.685 4.685 0 0 1-1.184 1.363 5.538 5.538 0 0 1 1.964 1.688c.512.71.768 1.61.768 2.701 0 1.628-.569 2.9-1.704 3.818-1.137.918-2.65 1.376-4.54 1.376H10v-17.66Zm29.006 0c1.925 0 3.572.442 4.942 1.325s2.056 2.25 2.056 4.103c0 1.091-.282 2.03-.846 2.818a5.715 5.715 0 0 1-2.172 1.83l4.345 7.584h-3.59l-3.747-6.752a14.611 14.611 0 0 1-1.223.052h-1.95v6.7h-3.356v-17.66ZM84.826 41v17.688h-3.585V41Zm-8.868 12.204-2.589.163c-1.324.082-2.015.643-2.015 1.532v.024c0 .923.762 1.473 1.945 1.473 1.535 0 2.659-.982 2.659-2.292zm-58.726-2.255h-3.876v4.96h3.642c.902 0 1.626-.207 2.172-.623.547-.416.82-1.004.82-1.766 0-.831-.252-1.467-.755-1.909-.503-.44-1.17-.662-2.003-.662Zm14.36-2.311c-.694 0-1.275.212-1.743.636-.468.424-.772 1.043-.91 1.857h5.28c-.173-.848-.485-1.476-.936-1.883-.451-.407-1.015-.61-1.691-.61Zm29.449 0c-.694 0-1.275.212-1.743.636-.468.424-.772 1.043-.91 1.857h5.28c-.173-.848-.485-1.476-.936-1.883-.452-.407-1.015-.61-1.691-.61ZM45.51 43.833h-2.081v5.402h2.133c1.006 0 1.838-.203 2.497-.61.66-.407.989-1.078.989-2.013 0-.987-.312-1.697-.937-2.13-.624-.432-1.491-.649-2.601-.649Zm-29.397 0h-2.757v4.39h2.497c.937 0 1.67-.178 2.199-.533.529-.355.793-.913.793-1.675 0-.831-.243-1.402-.728-1.714-.486-.312-1.154-.468-2.004-.468Z"/></svg>
```